src/test/kotlin/com/pubnub/extension/JsonElementTest.kt

```diff
@@ -110,21 +102,16 @@ class JsonElementTest {
         val mapper = MapperManager()
 
         // when
-        val (jsonElement, errorMessage) = objectUnderTest.tryDecryptMessage(cryptoModule, mapper)
```

Comment:
should we actually check that this is null?

Reply:
Good catch. I will change it.